input/blog/posts/_current-newsletter-draft.md

```diff
@@ -18,7 +18,35 @@ Welcome to the .NET Foundation update for January & February. Every other month
 
+### [NUnit](https://nunit.org)
 
+Many new releases are out at once, and work is starting on future major versions!
+
+#### [NUnit Framework 3.13](https://github.com/nunit/nunit/releases/tag/v3.13) and [3.13.1](https://github.com/nunit/nunit/releases/tag/v3.13.1)
+
+3.13 ships the first C# 8 Nullable Reference Types annotations! Almost all areas have been annotated. There are some important things to call out about this:
+
+- `Assert` methods don’t always throw exceptions when the assertion fails, due to the [Multiple Asserts](https://docs.nunit.org/articles/nunit/writing-tests/assertions/multiple-asserts.html) feature. This means that `Assert.NotNull(obj)` already didn’t mean that `obj` would never be null on the following line. `Assert.Catch` and the other `Assert` methods are the same way. We decided to not add nullability annotations unless they held true in every possible circumstance—a philosophy consistent with how .NET itself is annotated. This leads to the next point.
+
+- If you enable Nullable Reference Types in your test project, you will want to also add [NUnit.Analyzers 3.0](https://www.nuget.org/packages/NUnit.Analyzers/) or newer to your test project. Adding the analyzer is a great idea on its own, but it also specifically mitigates the problem in the previous point by suppressing C# compiler nullability warnings when we know it’s safe, when not in a Multiple Asserts context. This way you don’t have to use the C# null forgiveness operator (`!`) everywhere.
+
+Another addition in 3.13 is the [`FixtureLifeCycle`](https://docs.nunit.org/articles/nunit/writing-tests/attributes/fixturelifecycle.html) attribute. It enables you to stop sharing fixture instances between the tests inside that fixture. It is useful in combination with the [Parallelizable Attribute](https://docs.nunit.org/articles/nunit/writing-tests/attributes/parallelizable.html) because it allows tests within a fixture to run in isolation from each other without sharing instance fields and properties during parallel test runs.
+
+3.13 also fixes several issues running tests in .NET 5.0. If your tests target .NET 5.0, we recommend updating to this release or a newer one.
+
+#### [NUnit Console and Engine 3.12](https://github.com/nunit/nunit-console/releases/tag/v3.12)
+
+This release contains various improvements to running tests on .NET Core and Mono, and changes to extension loading logic to allow the Engine to better support extensions which target multiple platforms. There are additionally a number of fixes to issues that were identified with 3.12 Beta 1. Please also be aware that this will be the last version of the NUnit Engine to support .NET Standard 1.6.
+
+The .NET Core Console remains in beta ([release 3.12.0-beta2](https://github.com/nunit/nunit-console/releases/tag/v3.12.0-beta2)) due to some unresolved dependency loading and framework targeting issues—contributions to fix these issues would be very welcome.
```

Comment:
> contributions to fix these issues would be very welcome.

Might be worth calling this out up at the top? If anyone would like to get involved with NUnit as an OSS project, we have a perfect spot for them to dig in.

Reply:
Good thought. Is the new positioning what you had in mind?